R/environment.r

```diff
@@ -17,4 +17,26 @@ init_shadowenv <- function() {
     add_to_user_searchpath('.irk.get_shadowenv', get_shadowenv)
     add_to_user_searchpath('.irk.add_to_user_searchpath', add_to_user_searchpath)
 
+    # For the rest of the functions, please explain why the workaround is needed
+    # (=the problem) and link to the issue describing the problem.
+
+    # workaround for problems with vignette(xxx) not bringing up the vignette
+    # content in the browser: https://github.com/IRkernel/IRkernel/issues/267
+    add_to_user_searchpath('print.vignette', function(...){
+        utils:::print.vignette(...)
```

Comment:
yes, that’s why i mentioned it.

but we can use [this hack](https://github.com/IRkernel/repr/blob/df33b1574c8abb5eb7c19e7cc33fafbd60847e13/R/repr_help_files_with_topic.r#L12)